scripts/setup-k8s-master.sh.in

```diff
@@ -113,12 +117,28 @@ chown -R ubuntu:ubuntu /home/ubuntu/.kube
 #Install CockroachDB
 ####################
 
+LOCALITY="region={{Region}},availability-zone={{AvailabilityZone}}"
+ATTRS="instance-type={{InstanceType}}"
+
 # update default config to use desired settings
 /tmp/update-crdb-statefulset.py \
     https://cockroach-cloudformation.s3.amazonaws.com/kubernetes/scripts/cockroachdb-statefulset.yaml \
-    --size {{CockroachVolumeSize}} --version-tag {{CockroachDockerVersionTag}} > /tmp/cockroachdb-statefulset.yaml
+    --size {{CockroachVolumeSize}} --version-tag {{CockroachDockerVersionTag}} --locality $LOCALITY \
+    --attrs=$ATTRS > /tmp/cockroachdb-statefulset.yaml
 
 kubectl apply -f /tmp/cockroachdb-statefulset.yaml
 kubectl scale statefulset cockroachdb --replicas={{CockroachNodeCount}}
 kubectl create -f https://raw.githubusercontent.com/cockroachdb/cockroach/master/cloud/kubernetes/cluster-init.yaml
 kubectl create -f {{NodePortConfigUrl}}
+
+#create job to associate {{Region}} with {{RegionLatitude}}, {{RegionLongitude}}"
+
+if [[ {{CockroachDockerVersionTag}} =~ :v2 ]]
+then
+    kubectl run set-locality-coordinates --image=cockroachdb/cockroach --restart=OnFailure -- \
+    sql --insecure --host=cockroachdb-0.cockroachdb -e  \
+    "INSERT INTO system.locations (\"localityKey\", \"localityValue\", latitude, longitude) VALUES ('region', '{{Region}}', {{RegionLatitude}}, {{RegionLongitude}})"
```

Comment:
One question about error states here... I don't have enough context to know exactly when this is called, but maybe it would be useful to add `ON CONFLICT DO NOTHING` or `ON CONFLICT DO UPDATE ...`?

Reply:
Ah this is good to know for the future. This is called once. Just before the connection string is returned to the user, so this table will be empty.